cmd/openshift-tests/e2e.go

```diff
@@ -317,7 +317,8 @@ var staticSuites = testSuites{
 				if isDisabled(name) {
 					return false
 				}
-				return (strings.Contains(name, "[Suite:openshift/conformance/") && strings.Contains(name, "[sig-network]")) || isStandardEarlyOrLateTest(name)
+				// Skip new Netpol tests from K8's 1.21 for https://bugzilla.redhat.com/show_bug.cgi?id=1980141
+				return (strings.Contains(name, "[Suite:openshift/conformance/") && strings.Contains(name, "[sig-network]")) && !strings.Contains(name, "Netpol") || isStandardEarlyOrLateTest(name)
```

Comment:
You could make your new addition an if block up above this line?  Putting it all in the return is harder to read

https://github.com/openshift/origin/pull/26316/files#diff-510a5c561f86648198a7020f53ef92765ba67897d8806c0ceacd6460255d7f60R103

Reply:
Done